legality.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'activesupport', '~> 4.1'
+  spec.add_dependency 'activesupport', '~> 5.2'
```

Comment:
I think this should be `>= 4.1` rather than explicitly requiring `5.2`. There's nothing about this gem that actually requires Rails 5.